src/objects-table/ObjectsTable.js

```diff
@@ -131,7 +133,7 @@ class ObjectsTable extends React.Component {
             sorting: this.props.initialSorting,
             searchValue: null,
             detailsObject: null,
-            selection: new Set(),
+            selection: new Set(this.props.initialSelection),
```

Comment:
Let's say I pass an ID in `initialSelection` which is not in fact in allObjects, the messages about the selection won't be accurate. Is this a problem?

Reply:
Not really, as we might have them hidden by the filters but that are in fact selected